wfe/web-front-end.go

```diff
@@ -1045,7 +1044,7 @@ func (wfe *WebFrontEndImpl) BuildID(response http.ResponseWriter, request *http.
 
 	response.Header().Set("Content-Type", "text/plain")
 	response.WriteHeader(http.StatusOK)
-	detailsString := fmt.Sprintf("Boulder=(%s %s) Golang=(%s) BuildHost=(%s)", core.GetBuildID(), core.GetBuildTime(), runtime.Version(), core.GetBuildHost())
+	detailsString := fmt.Sprintf("Boulder=(%s %s)", core.GetBuildID(), core.GetBuildTime())
```

Comment:
Well, if we leave it in for development builds (which I assume includes letsencrypt-demo.org since it was apparently added to help Kuba and other client folks), we will still leak information about our build environment. I'm not sure filtering it from letsencrypt-demo.org would be good for them.

Tangentially, I would much rather we do all of our filtering and such inside of wfe where we can all know what our security model is. Having stuff leak out to other config settings means we won't have all that info in one repo, and in one team of folks, but instead spread out, making it easier for mistakes to happen in the future. Mistakes like removing the filtering accidentally, or, in the opposite direction, leaving the filtering in there after it's been removed from the wfe to accrete as tech debt.